src/routes/guides/building-a-data-source/+page.svelte

```diff
@@ -464,10 +450,7 @@ content-length: 95
     the Available data sources section and it will no longer require an <em>X-Preview-Token</em> header
     to be queried.
   </p>
-  <SectionTitle
-    title="Next steps
-"
-  />
+  <SectionTitle title="Next steps" />
   <p>
     If you created the data source definition under your own <em>test/</em>
     <your-own-name>
```

Comment:
This is supposed to be `test/<your-own-name>`, now there's some extra `</your-own-name>` couple of rows down and this top one is interpreted as some unknown tag.

Reply:
![image](https://github.com/user-attachments/assets/afba6374-b1a6-4719-9367-708084dc7490)
This might reveal the issue better than GitHub rendering it.

Reply:
I took the freedom to fix this and push it.